docs/mine-hnt/validators/troubleshoot.mdx

```diff
@@ -0,0 +1,36 @@
+---
+id: validator-troubleshooting
+title: Troubleshooting
+hide_title: true
+sidebar_label: Troubleshooting
+slug: /mine-hnt/validators/validator-troubleshooting
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Troubleshooting
+
+There is a ```#validator``` [Discord](https://discord.gg/helium)  channel available to reach other testnet participants. Also use this channel to share feedback and report issues.
+
+## View logs
+This is always helpful to get an idea of what's going on:
+
+```docker exec miner tail -F /var/log/miner/console.log```
```

Comment:
Should be `docker exec validator tail -F /var/log/console.log`

Reply:
fixed, thanks